test/spdm-responder-emu/src/main.rs

```diff
@@ -25,10 +36,104 @@ use spdm_emu::socket_io_transport::SocketIoTransport;
+                            size_of::<usize>() * 3 + // send_message in spdm_emu
+                            RECEIVER_BUFFER_SIZE + // receive_secured_message
+                            size_of::<usize>() * 3 + // receive_secured_message
+                            RECEIVER_BUFFER_SIZE + // decode_secured_message
+                            RECEIVER_BUFFER_SIZE + // decode_secured_message
+                            size_of::<usize>() * 7 + // decode_secured_message
+                            RECEIVER_BUFFER_SIZE + // decode_msg
+                            size_of::<usize>() * 10 + // decode_msg
+                            size_of::<Reader>() * 2 + // decode_msg
+                            size_of::<SpdmDigestStruct>() + // decode_msg
+                            size_of::<usize>() * 4 + // receive_message in spdm_emu
+                            size_of::<Reader>() + // receive_message in spdm_emu
+                            size_of::<SpdmSocketHeader>() + // receive_message in spdm_emu
+                            RECEIVER_BUFFER_SIZE + // receive in socket_io_transport
+                            size_of::<usize>() * 5 + // receive in socket_io_transport
+                            // handle_spdm_measurement
+                            MAX_SPDM_MSG_SIZE + // send_receive_spdm_measurement_record
+                            size_of::<Writer>() + // send_receive_spdm_measurement_record
+                            size_of::<Reader>() + // write_spdm_measurement_response
+                            size_of::<SpdmMessageHeader>() + // write_spdm_measurement_response
+                            size_of::<SpdmGetMeasurementsRequestPayload>() + // write_spdm_measurement_response
+                            size_of::<SpdmMessage>() + // write_spdm_measurement_response
+                            size_of::<SpdmMeasurementsResponsePayload>() + // write_spdm_measurement_response
+                            size_of::<Writer>() + // write_spdm_measurement_response
+                            SPDM_NONCE_SIZE + // write_spdm_measurement_response
+                            MAX_SPDM_MEASUREMENT_VALUE_LEN * 255 + // ~ measurement_collection
+                            MAX_SPDM_MEASUREMENT_RECORD_SIZE * 255 + // ~ measurement_collection
+                            size_of::<ManagedBuffer12Sign>() + // generate_measurement_signature
+                            size_of::<SpdmSignatureStruct>() + // generate_measurement_signature
+                            SENDER_BUFFER_SIZE + // send_secured_message
+                            size_of::<usize>() * 2 + // send_secured_message
+                            SENDER_BUFFER_SIZE + // encode_secured_message
+                            SENDER_BUFFER_SIZE + // encode_secured_message
+                            size_of::<usize>() * 7 + // encode_secured_message
+                            SENDER_BUFFER_SIZE + // encode_msg
+                            size_of::<usize>() * 13 + // encode_msg
```

Comment:
there are too many details. Please simplify that.

Reply:
> there are too many details. Please simplify that.

simplified on latest commit.